Classes/DoorbellDialog.m

```diff
@@ -42,7 +42,11 @@ - (id)initWithFrame:(CGRect)frame
         boxFrame.size = CGSizeMake(300, 255);
         boxFrame = [self calculateNewBoxFrame:boxFrame];
         _boxView = [[UIView alloc] initWithFrame:boxFrame];
-        _boxView.backgroundColor = [UIColor systemBackgroundColor];
+        if (@available(iOS 13.0, *)) {
+            _boxView.backgroundColor = [UIColor systemBackgroundColor];
+        } else {
+            _boxView.backgroundColor = [UIColor whiteColor];
+        }
```

Comment:
I considered putting this in a new `Colours` file, but the project has a very simple project structure, so I decided to keep it that way. It does mean that we have a few annoying `if else` statements in this file.

Reply:
Definitely happy keeping the simpler structure!